frontend/packages/dev-console/src/components/monitoring/overview/MonitoringMetricsSection.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import {
+  Accordion,
+  AccordionItem,
+  AccordionToggle,
+  AccordionContent,
+  Grid,
+  GridItem,
+} from '@patternfly/react-core';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { requirePrometheus } from '@console/internal/components/graphs';
+import MonitoringDashboardGraph from '../dashboard/MonitoringDashboardGraph';
+import { workloadMetricQueries } from './queries';
+import './MonitoringSection.scss';
+
+const WorkloadGraphs = requirePrometheus(({ resource }) => {
+  const ns = resource.metadata.namespace;
+  const workloadName = resource.metadata.name;
+  const workloadType = resource.kind.toLowerCase();
+  return (
+    <>
+      <Grid className="co-m-pane__body">
+        {_.map(workloadMetricQueries, (q, i) => (
+          <GridItem span={12} key={i}>
+            <MonitoringDashboardGraph
+              title={q.title}
+              namespace={ns}
+              graphType={q.chartType}
+              query={q.query({ ns, workloadName, workloadType })}
+              humanize={q.humanize}
+              byteDataType={q.byteDataType}
+            />
+          </GridItem>
+        ))}
+      </Grid>
+    </>
+  );
+});
+
+type MonitoringMetricsSectionProps = {
```

Comment:
```suggestion
export interface MonitoringMetricsSectionProps = {
```

Reply:
I don't think we need to export props, for one file we should have just one either `interface` or `type`